nss/src/group/mod.rs

```diff
@@ -1,33 +1,33 @@
 use crate::{info, REQUEST_TIMEOUT};
 use libc::gid_t;
-use libnss::group::{Group, GroupHooks};
+use libnss::group::{Group as NSSGroup, GroupHooks};
 use libnss::interop::Response;
 use tokio::runtime::Builder;
 use tonic::Request;
 
 use crate::client::{self, authd};
-use authd::GroupEntry;
+use authd::Group;
```

Comment:
i.e. `use authd::Group as AuthdGroup;`

Reply:
We already define an `AuthdGroup` struct in this file :/

Reply:
> Then we could also keep the group_to_group_entry function name.

Or `authd_group_to_group_entry` if you prefer that even if we don't import `authd::Group` as `AuthdGroup`.

Reply:
> We already define an AuthdGroup struct in this file :/

Oh, yeah... But maybe we should update the hook struct then? I think it makes more sense, tbh. Maybe rename the struct to `GroupHook` (for example), as it's only used to implement the functions and then "link" them in `lib.rs`

Reply:
I renamed it to `AuthdGroupHooks` (and the other hooks defined in the NSS module too)